tests/Radiation/MeterRadiationTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Sundata\Utilities\Test\Radiation;
+
+use Generator;
+use PHPUnit\Framework\TestCase;
+use Sundata\Utilities\Radiation\MeterRadiation;
+use Sundata\Utilities\Radiation\MeterRadiationRequest;
+
+class MeterRadiationTest extends TestCase
+{
+    public function MeterRadiationDataProvider(): Generator
+    {
+        yield 'South facing 80 degrees' => [
```

Comment:
tippie; verwoord de testcases als requirements. e.g.;
South facing w 80 degrees angle should give high radiation

iig iets van de verwachting er in dus. dat maakt teruglezen makkelijker